spec/micro/README.md

```diff
@@ -0,0 +1,381 @@
+
+Properties of Connection:
+
+- Verifies the validity of the counterparty client
+- Establishes a unique identifier on each side for a shared abstract understanding (the connection)
+- Establishes an agreement on the IBC version and supported features
+- Allows multiple connections to be built against the same client pair
+- Establishes the delay period so this security parameter can be instantiated differently for different connections against the same client pairing.
+- Defines which channel orderings are supported
+
+Properties of Channel:
+
+- Separates applications into dedicated 1-1 communication channels. This prevents applications from writing into each other's channels.
+- Allows applications to come to agreement on the application parameters (version negotiation). Ensures that each side can understand the other's communication and that they are running mutually compatible logic. This version negotiation is a multi-step process that allows the finalized version to differ substantially from the one initially proposed
+- Establishes the ordering of the channel
+- Establishes unique identifiers for the applications on either chain to use to reference each other when sending and receiving packets.
+- The application protocol can be continually upgraded over time by using the upgrade handshake which allows the same channel which may have accumulated state to use new mutually agreed upon application packet data format(s) and associated new logic.
+- Ensures exactly-once delivery of packet flow datagrams (Send, Receive, Acknowledge, Timeout)
+- Ensures valid packet flow (Send => Receive => Acknowledge) XOR (Send => Timeout)
+
+### Identifying Counterparties
+
+In core IBC, the connection and channel handshakes serve to validate the clients are valid clients of the counterparty, ensure the IBC version and application versions are mutually compatible, as well as providing unique identifiers for each side to refer to the counterparty.
+
+Since we are removing handshakes in IBC lite, we must have a different way to provide the chain with knowledge of the counterparty. With a client, we can prove any key/value path on the counterparty. However, without knowing which identifier the counterparty uses when it sends messages to us; we cannot differentiate between messages sent from the counterparty to our chain vs messages sent from the counterparty with other chains. Most implementations will not be able to store the ICS-24 paths directly as a key in the global namespace; but will instead write to a reserved, prefixed keyspace so as not to conflict with other application state writes. Thus the counteparty information we must have includes both its identifier for our chain as well as the key prefix under which it will write the provable ICS-24 paths.
+
```

Comment:
I feel like adding a simple diagram of the above will aid significantly in understanding, especially given how critical this is to part of the design.